[2025-03-28T02:15:10.859+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-28T02:15:11.514+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_dbt_clean_reviews_pipeline.dbt_run_transactional scheduled__2024-03-28T00:00:00+00:00 [queued]>
[2025-03-28T02:15:11.633+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_dbt_clean_reviews_pipeline.dbt_run_transactional scheduled__2024-03-28T00:00:00+00:00 [queued]>
[2025-03-28T02:15:11.635+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-28T02:15:12.682+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run_transactional> on 2024-03-28 00:00:00+00:00
[2025-03-28T02:15:14.394+0000] {standard_task_runner.py:72} INFO - Started process 187415 to run task
[2025-03-28T02:15:14.638+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'load_dbt_clean_reviews_pipeline', 'dbt_run_transactional', 'scheduled__2024-03-28T00:00:00+00:00', '--job-id', '2001', '--raw', '--subdir', 'DAGS_FOLDER/dag_load_reviews.py', '--cfg-path', '/tmp/tmpyyr0_hdc']
[2025-03-28T02:15:14.640+0000] {standard_task_runner.py:105} INFO - Job 2001: Subtask dbt_run_transactional
[2025-03-28T02:15:16.351+0000] {task_command.py:467} INFO - Running <TaskInstance: load_dbt_clean_reviews_pipeline.dbt_run_transactional scheduled__2024-03-28T00:00:00+00:00 [running]> on host DESKTOP-IM7V4RI.
[2025-03-28T02:15:18.568+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohammed' AIRFLOW_CTX_DAG_ID='load_dbt_clean_reviews_pipeline' AIRFLOW_CTX_TASK_ID='dbt_run_transactional' AIRFLOW_CTX_EXECUTION_DATE='2024-03-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-28T00:00:00+00:00'
[2025-03-28T02:15:18.707+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-28T02:15:18.707+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-28T02:15:18.708+0000] {logging_mixin.py:190} INFO - Current task name:dbt_run_transactional state:running start_date:2025-03-28 02:15:11.515444+00:00
[2025-03-28T02:15:18.709+0000] {logging_mixin.py:190} INFO - Dag name:load_dbt_clean_reviews_pipeline and current dag run status:running
[2025-03-28T02:15:18.710+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-28T02:15:18.740+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-28T02:15:18.742+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/mohammed/.dbt/dbt_projects/bank_reviews_transactional && dbt run']
[2025-03-28T02:15:18.883+0000] {subprocess.py:99} INFO - Output:
[2025-03-28T02:15:20.605+0000] {job.py:229} INFO - Heartbeat recovered after 11.99 seconds
[2025-03-28T02:15:50.471+0000] {job.py:229} INFO - Heartbeat recovered after 21.73 seconds
[2025-03-28T02:15:58.038+0000] {job.py:229} INFO - Heartbeat recovered after 23.33 seconds
[2025-03-28T02:16:09.406+0000] {subprocess.py:106} INFO - [0m02:16:09  Running with dbt=1.9.3
[2025-03-28T02:16:12.694+0000] {subprocess.py:106} INFO - [0m02:16:12  Registered adapter: postgres=1.9.0
[2025-03-28T02:16:17.026+0000] {subprocess.py:106} INFO - [0m02:16:17  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-28T02:16:17.028+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-03-28T02:16:17.028+0000] {subprocess.py:106} INFO - - models.bank_reviews_transactional.transactional
[2025-03-28T02:16:18.572+0000] {subprocess.py:106} INFO - [0m02:16:18  Found 1 model, 1 source, 433 macros
[2025-03-28T02:16:18.681+0000] {subprocess.py:106} INFO - [0m02:16:18
[2025-03-28T02:16:18.703+0000] {subprocess.py:106} INFO - [0m02:16:18  Concurrency: 4 threads (target='dev')
[2025-03-28T02:16:18.705+0000] {subprocess.py:106} INFO - [0m02:16:18
[2025-03-28T02:16:52.877+0000] {job.py:229} INFO - Heartbeat recovered after 38.18 seconds
[2025-03-28T02:17:07.144+0000] {subprocess.py:106} INFO - [0m02:17:07  1 of 1 START sql table model transactional.clean_reviews ....................... [RUN]
[2025-03-28T02:17:11.933+0000] {job.py:229} INFO - Heartbeat recovered after 25.49 seconds
[2025-03-28T02:17:12.021+0000] {subprocess.py:106} INFO - [0m02:17:12  1 of 1 OK created sql table model transactional.clean_reviews .................. [[32mSELECT 153[0m in 4.81s]
[2025-03-28T02:17:12.414+0000] {subprocess.py:106} INFO - [0m02:17:12
[2025-03-28T02:17:12.422+0000] {subprocess.py:106} INFO - [0m02:17:12  Finished running 1 table model in 0 hours 0 minutes and 53.72 seconds (53.72s).
[2025-03-28T02:17:12.731+0000] {subprocess.py:106} INFO - [0m02:17:12
[2025-03-28T02:17:12.753+0000] {subprocess.py:106} INFO - [0m02:17:12  [32mCompleted successfully[0m
[2025-03-28T02:17:12.767+0000] {subprocess.py:106} INFO - [0m02:17:12
[2025-03-28T02:17:12.772+0000] {subprocess.py:106} INFO - [0m02:17:12  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-28T02:17:16.253+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-28T02:17:20.483+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-28T02:17:20.527+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=load_dbt_clean_reviews_pipeline, task_id=dbt_run_transactional, run_id=scheduled__2024-03-28T00:00:00+00:00, execution_date=20240328T000000, start_date=20250328T021511, end_date=20250328T021720
[2025-03-28T02:17:21.917+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-28T02:17:21.918+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-28T02:17:21.918+0000] {logging_mixin.py:190} INFO - Dag name:load_dbt_clean_reviews_pipeline queued_at:2025-03-28 02:14:52.647768+00:00
[2025-03-28T02:17:21.919+0000] {logging_mixin.py:190} INFO - Task hostname:DESKTOP-IM7V4RI. operator:BashOperator
[2025-03-28T02:17:22.259+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-28T02:17:22.717+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-03-28T02:17:23.650+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
